src/libraries/System.IO.Pipes/tests/NamedPipeTests/NamedPipeTest.CreateClient.cs

```diff
@@ -17,13 +17,15 @@ public static void NullPipeName_Throws_ArgumentNullException()
         {
             AssertExtensions.Throws<ArgumentNullException>("pipeName", () => new NamedPipeClientStream(null));
             AssertExtensions.Throws<ArgumentNullException>("pipeName", () => new NamedPipeClientStream(".", null));
+            AssertExtensions.Throws<ArgumentNullException>("pipeName", () => new NamedPipeClientStream(".", null, PipeAccessRights.FullControl, PipeOptions.None, TokenImpersonationLevel.None, HandleInheritability.None));
```

Comment:
Do these tests run on Unix? I'd expect such use would result in a PlatformNotSupportedException being thrown, since the Unix impl is hardcoded to always do that and not do any arg validation.

Reply:
Thanks for catching that. I broke these out into separate platform-specific tests.